rollup.config.js

```diff
@@ -1,83 +1,54 @@
-import project from "./package.json";
-import babel from "rollup-plugin-babel";
-import { uglify } from "rollup-plugin-uglify";
+import project from './package.json';
+import buble from '@rollup/plugin-buble';
+import { terser } from 'rollup-plugin-terser';
+
+function buildConfig({ dest, format, transpile = true }) {
```

Comment:
Maybe something like `filename` instead of `dest` makes more sense.

Reply:
Changed.